packages/agent/tests/dwn-permissions-util.spec.ts

```diff
@@ -106,19 +106,21 @@ describe('DwnPermissionsUtil', () => {
 
       const protocol = 'http://example.com/protocol';
 
-      await GrantsUtil.createRecordsGrants({
+      const deviceXRecordGrants = await GrantsUtil.createRecordsGrants({
```

Comment:
Remove the reliance on `fetchGrants` from this test, as it's out of scope for the util.